packages/x-date-pickers-pro/src/SingleInputDateRangeField/SingleInputDateRangeField.tsx

```diff
@@ -28,7 +25,7 @@ type DateRangeFieldComponent = (<TDate>(
  */
 const SingleInputDateRangeField = React.forwardRef(function SingleInputDateRangeField<TDate>(
   inProps: SingleInputDateRangeFieldProps<TDate>,
-  ref: React.Ref<HTMLDivElement>,
+  inRef: React.Ref<HTMLDivElement>,
```

Comment:
Nitpick: Was there a need to rename the prop? I can't see a naming conflict anywhere. 🙈

Reply:
👌 Just confirming that this is probably the case in most if not all field components. 😉